src/us/kbase/workspace/kbase/InitWorkspaceServer.java

```diff
@@ -516,7 +516,7 @@ private static AbstractHandleClient getHandleServiceClient(
 						"https is recommended.");
 				cli.setIsInsecureHttpConnectionAllowed(true);
 			}
-			cli.isOwner(Arrays.asList("FAKEHANDLE_-100"));
```

Comment:
@MrCreosote HS2 is attempting to search for number after 'KBH'. In this case, it does a `int('FAKEHANDLE_-100')` and errors. Should I make it ok to pass any arbitrary handle_id?

Reply:
How does the old service handle it?

Reply:
seems like it's just return false

Reply:
So if it sees a namespace it doesn't recognize it returns false?

Actually, here's the code: https://github.com/kbase/handle_service/blob/master/lib/Bio/KBase/AbstractHandle/AbstractHandleImpl.pm#L1311-L1320

So it'll try and strip off the namespace, fail to do so, and then not find anything.

That's not a great API in my opinion, but now's not the time to try and fix it. What does your code do?

Reply:
Or maybe now is the time to try and fix it. HS2 could throw an error and we could either check for that error and consider it a successful start or we could pass another handle ID where it's reasonable to return false (like KBH_-1 or something)

Reply:
@MrCreosote I updated HS2 to return false if handle_id is not found in the DB.

Reply:
But the DB only has the number part of the handle ID, right? so if KBH_42 exists and I send FAKE_42 will that return true?

Reply:
@MrCreosote no. it is doing a split('KBH_'). 'FAKE_42' will return false.

Reply:
Ok, so it does the same thing as the old service then? It's searching a set of integers for 'FAKE_42'?

Reply:
yup. yup.

Reply:
OK. I can live with that but we have a chance to make a better interface, I think. Up to you.

Reply:
cool. I think I am pretty happy with what it is now. I think it's reasonable to return false for `isOwner` if it's a fake handle id.